src/chrome/content/rules/LinuxFoundation.xml

```diff
@@ -20,18 +20,11 @@
 
 	Partially covered hosts in *linuxfoundation.org:
 
-		- collabprojects ʰ
 		- events ʰ
-		- eventsstg ʰ
 		- www ʰ
```

Comment:
I removed `www` in https://github.com/EFForg/https-everywhere/pull/10537/commits/d16b3613b69abb6d8f35ae28c3cb15cb37e1d0d1.